boxoffice/assets/js/views/org.js

```diff
@@ -2,30 +2,88 @@
 var Ractive = require('ractive');
 import {eventBus} from './main_admin.js'
 var NProgress = require('nprogress');
-import {fetch, urlFor, setPageTitle} from '../models/util.js';
+import {Util, fetch, post, getFormParameters, urlFor, setPageTitle} from '../models/util.js';
 import {orgTemplate} from '../templates/org.html.js';
 import {SideBarView} from './sidebar.js'
 
 export const OrgView = {
   render: function({org_name}={}) {
+    const DEFAULT = {
+      showForm: true,
+      hideForm: false,
+      btnDisable: true,
+      btnEnable: false,
+      empty: ""
+    };
+
     fetch({
       url: urlFor('view', {resource: 'o', id: org_name, root: true})
-    }).then(function({id, org_title, item_collections}){
+    }).then(function({id, org_title, item_collections, form}) {
+      let ICForm = Ractive.extend({
+        isolated: false,
+        template: Util.getFormTemplate(form),
+        data: {
+          formId: Util.getElementId(form),
+        },
+        onFormSubmit: function(event) {
+          event.original.preventDefault();
+          let self = this;
```

Comment:
Any reason why `self` is required here? This shouldn't be required if you use the arrow function instead?

Reply:
Fixed